src/components/PatchesPage/PatchCard.tsx

```diff
@@ -73,7 +73,7 @@ export const PatchCard: React.VFC<Props> = ({
         <strong>{projectIdentifier}</strong>
       </StyledRouterLink>
     ) : (
-      `${projectMetadata.owner}/${projectMetadata.repo}`
+      `${projectMetadata?.owner}/${projectMetadata?.repo}`
```

Comment:
While this temporarily alleviates the issue it doesn't get at the real cause. This results in these values being undefined. 
<img width="881" alt="image" src="https://user-images.githubusercontent.com/4605522/215895480-82742b13-5cb5-46aa-a9de-ae42269222c5.png">

Ideally, we should  try to understand why these values are undefined in the first place. 

Looks like we are missing both  `projectIdentifier` and `projectMetadata`.